python/ray/dashboard/dashboard.py

```diff
@@ -701,6 +711,325 @@ def run(self):
             if counter % 10:
                 self.update_nodes()
 
+class TuneCollector(threading.Thread):
+
+    EXPR_META_FILE = "trial_status.json"
+    JOB_META_FILE = "job_status.json"
+    EXPR_RESULT_FILE = "result.json"
+    EXPR_PARAM_FILE = "params.json"
+
+    def __init__(self, logdir, logger, reload_interval):
+        """Initialize collector worker thread.
+        Args
+            logdir (str): Directory path to save the status information of
+                          jobs and trials.
+            logger (Logger): Logger for collector thread.
+        """
+
+        super().__init__()
+        self._logdir = DEFAULT_RESULTS_DIR
+        self._monitored_jobs = set()
+        self._monitored_trials = set()
+        self._result_offsets = {}
+        self._trial_records = {}
+        self._result_records = {}
+        self._job_records = {}
+        self.logger = logger
+        self._data_lock = threading.Lock()
+        self._reload_interval = reload_interval
+        self._loss_attributes = []
+
+    @classmethod
+    def parse_json(cls, json_file):
```

Comment:
Let's use https://ray.readthedocs.io/en/latest/tune-usage.html#analyzing-results to direct from analysis and parse the try to render the `analysis.dataframe()`